pkg/ingest/setup_test.go

```diff
@@ -3,11 +3,94 @@ package ingest
 import (
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gstruct"
 	v1 "github.com/solo-io/build/pkg/api/v1"
 )
 
 var _ = Describe("ingest config", func() {
 
+	const relativePathToSampleConfig = "../../examples/sample-solo-project.yaml"
+	var buildId = "some-build-id"
+	const nonReleaseTagValue = ""
+	var explicitBuildEnvVars = &v1.BuildEnvVars{
+		BuildId:       buildId,
+		TaggedVersion: nonReleaseTagValue,
+	}
+	var sampleConfigIngested = v1.BuildRun{
```

Comment:
nit: this isn't the ingested config, would refer to this as "expectedConfig" or something similar

Reply:
fixed